style.css

```diff
@@ -114,20 +114,27 @@ body {
   border-radius: 16px;
   background-color: #f7f7f9;
   align-items: center;
+  justify-content: center;
   padding-top: 100px;
-  gap: 30px;
+  gap: 40px;
   margin-bottom: 20px;
-  width: 100%;
+  width: auto;
+  height: 2788px;
+  padding-bottom: 150px;
```

Comment:
[OPTIONAL]
- [ ] Kindly reduce the `padding-bottom` of the "works" section on the mobile version, the space at the bottom of the section is too big compared to the figma design.
### Your design :point_down: 
![Screenshot from 2023-05-06 13-46-28](https://user-images.githubusercontent.com/91142432/236628176-1aa4bf0e-681b-4cc1-a7f3-31f354967e25.png)
### What it supposed to look like :point_down: 
![Screenshot from 2023-05-06 13-48-08](https://user-images.githubusercontent.com/91142432/236628216-de2a71b3-5d29-451f-87e8-e36a42281d24.png)

_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_